pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+name = "mammosunits"
+version = "0.1.0"
+description = "A short description of your project"
+authors = [{ name = "Sam Holt", email = "samjrholt@gmail.com" },
+           { name = "Hans Fangohr", email = "hans.fangohr@mpsd.mpg.de"} ]
+requires-python = ">=3.9"
+dependencies = [
+    "pytest",
+    "astropy>=7.0.0",
+    "jupyter",
+]
+
+[build-system]
+requires = ["setuptools", "wheel"]
```

Comment:
I am not sure if wheel is required here? However, it can be, so good to check!

Reply:
Should not be required.

Reply:
I think PEP 517 requires it? We have it in our other packages :)